bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -1151,8 +1151,8 @@
     </changeSet>
     <changeSet id="bahmni-core-201906101318" author="Vinay, Indraneel, Samuel34">
         <preConditions onFail="MARK_RAN">
-        	<sqlCheck expectedResult="0">
-        		select count(*) from global_property where property = 'order.durationUnitsConceptUuid';
+        	<sqlCheck expectedResult="1">
```

Comment:
you need to have the changeset id changed. otherwise, liquibase will throw error for SHA being different if this changeset already was executed once, which will be the case as this is in the core.

Reply:
https://forum.liquibase.org/t/preconditions-affect-checksum/959